dist/angular-svg-radial.js

```diff
@@ -0,0 +1,36 @@
+/*!
+ * angular-svg-radial 0.1.0
+ * (c) 2015 Vokal LLC
+ * https://github.com/vokal/angular-svg-radial
+ */
+angular.module("svgRadial", []).directive("radialProgress", function() {
+    "use strict";
+    return {
+        restrict: "E",
+        transclude: true,
+        replace: true,
+        template: '<div class="radial-progress">' + '<div class="base"></div>' + '<div class="radial">' + '<div class="piece"></div>' + '<div class="piece"></div>' + "</div>" + '<div class="transclude" ng-transclude></div>' + "</div>",
```

Comment:
Is there any point in having a doc-stripped version, though?  You're either including an expanded file so you can debug, or you're done and dropping in the minified one.  The terseness of the expanded file is irrelevant, it's only there so you can debug.  Just drop in the doc-filled source file for those cases.

0.x.x releases are by definition unstable, there's no promise of non-breaking changes until you get to v1.  From there, breaking changes must iterate the major version.  Until then, break whatever/whenever you want.